docs/application/web/api/5.0/device_api/mobile/tizen/alarm.html

```diff
@@ -887,10 +885,13 @@ <h4 id="AlarmAbsolute::constructor">Constructors</h4>
 <a class="backward-compatibility-anchor" name=""></a><code><b><span class="methodName">Constructor (Date, <a href="#ByDayValue">ByDayValue</a>[])</span></b></code>
 </dt>
 <pre class="webidl prettyprint">AlarmAbsolute(Date date, <a href="#ByDayValue">ByDayValue</a>[] daysOfTheWeek);</pre>
-<dt class="method" id="">
+<dt class="extendedattribute deprecated" id="">
 <a class="backward-compatibility-anchor" name=""></a><code><b><span class="methodName">Constructor (Date, long)</span></b></code>
 </dt>
-<pre class="webidl prettyprint">AlarmAbsolute(Date date, long period);</pre>
+<pre class="webidl prettyprint"><span class="nocode deprecated">AlarmAbsolute(Date date, long period);</span></pre>
+<p class="deprecated"><b>Deprecated.</b>
+ Since Tizen 4.0 constructor AlarmAbsolute(Date date, long period) is deprecated.
+              </p>
 </dl>
```

Comment:
Question. In this page we have below sentence.

(e.g. recursive AlarmAbsolute, created from constructor AlarmAbsolute(Date date, long period) is passed).

Is it ok?

Reply:
Regarding deprecation policy https://docs.tizen.org/application/web/tutorials/deprecation-policy, API deprecated in 4.0 is still valid on 5.0 and become unavailable since next version 5.5.
Because of this we left mentioned comment in 5.0 documentation, but it is removed in 5.5.